src/certificates/certificates-card/certificate-signatories/signatory/Signatory.jsx

```diff
@@ -106,8 +126,16 @@ const Signatory = ({
           </Form.Group>
         </Stack>
       ) : (
-        null
-        // <Stack direction="horizontal" gap="2" className="stack-container" data-testid="signatory-view" /> TODO https://youtrack.raccoongang.com/issue/AXIMST-166
+        <Stack direction="horizontal" gap="2" className="stack-container" data-testid="signatory-view">
+          <Stack className="text-stack">
+            <p><b>{intl.formatMessage(messages.nameLabel)}</b> {name}</p>
+            <p><b>{intl.formatMessage(messages.titleLabel)}</b> {title}</p>
+            <p><b>{intl.formatMessage(messages.organizationLabel)}</b> {organization}</p>
+          </Stack>
+          <div className="image-container">
+            {signatureImagePath && <img src={`${getConfig().STUDIO_BASE_URL}${signatureImagePath}`} alt="signature" className="d-block w-100" />}
```

Comment:
[nit]: Let's add a translation for the alt text and try to use Paragon component Image